schedule/yast/sle/guided_ext4/guided_ext4_x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name:           guided_ext4
+description:    >
+  Guided Partitioning installation with ext4 filesystem.
+vars:
+  FILESYSTEM: ext4
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  guided_partitioning:
```

Comment:
I guess there is a reason, but why the change ? In guided_btrfs and guided_ext4 we don't have these extra steps. I mean, it works so it's ok, but I would just have copied the same.

Reply:
you mean: select_guided_setup,accept_default_part_scheme,accept_default_fs_options in [guided_btrfs](https://openqa.nue.suse.com/tests/10436392) , [guided_xfs](https://openqa.nue.suse.com/tests/10436394) and [guided_ext4](https://openqa.nue.suse.com/tests/10436393) they all have these 3 modules.

Reply:
Now I'm really confused: where are those modules called? for example in btrfs, those modules are not in the [schedule file](https://openqa.nue.suse.com/tests/10436392/settings/schedule/yast/sle/guided_btrfs/guided_btrfs.yaml) but also not in the [default](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/yast/sle/flows/default.yaml). Do you know hat is going on ?

Reply:
So correct me if I'm wrong, but you ned to remove YAML_SCHEDULE_FLOWS from the VRs in order to use the new schedule file. I re-ran the VRs for btrfs, ext4 and xfs with YAML_SCHEDULE_FLOWS=''